scalableminds/webknossos
# Improve import fix onboarding redirect

### URL of deployed dev instance (used for testing):
- https://improveimportfixonboardingredirect.webknossos.xyz

### Steps to test:
- upload and import a dataset via the usual flow
- visit `/onboarding` and upload a dataset in the third step (if you change the state initialization in `onboarding.js` to `currentStep: 2` you can skip the org/admin creation). the import view should show up as a modal

### Issues:
- follow up to #2851


------
